src/api/routes/v1/utils.ts

```diff
@@ -26,6 +26,27 @@ export function formatAgreementScore(agreement: AgreementScore): {
   }
 }
 
+/**
+ * Formats amendments from string to list.
+ *
+ * @param amendmentsDb - Amendments string from database.
+ * @returns A list of amendments.
+ */
+export async function formatAmendments(
+  amendmentsDb: string,
+): Promise<Array<{ id: string; name: string }>> {
+  const res: Array<{ id: string; name: string }> = []
+  const amendmentsList = amendmentsDb.split(',')
+  amendmentsList.forEach(async (amendment) => {
+    const info = await query('amendments_info')
+      .select('id', 'name')
+      .where('id', amendment)
+      .first()
```

Comment:
Why return both the ID and the name?

Reply:
One is the pretty name the other is the id.

Reply:
Yes, but why are both needed here?

Reply:
It would be for the API for validators to show the list of both amendment_id and name. Also, it would be for the Voting Tab on the explorer for validator as you would not need to search for name by id there, just need to make one API call. In VHS db ballot, each validator has a field amendments which is a string of ids comma separated, which is why this function is needed.